### PR TITLE
added archiveLink in file model and changed download link in agenda-a…

### PR DESCRIPTION
**Pull request by StuyckRuben**
…ctions
https://kanselarij.atlassian.net/browse/KAS-3939

- [x] Merge https://github.com/kanselarij-vlaanderen/app-kaleidos/pull/405
- [x] Merge https://github.com/kanselarij-vlaanderen/file-bundling-service/pull/13

### CODE REVIEW

File: app/components/agenda/agenda-header/agenda-actions.js [hunk 1]
```diff
@@ -306,7 +306,7 @@ export default class AgendaAgendaHeaderAgendaActions extends Component {
         if (job.status === job.SUCCESS) {
           const url = await fileDownloadUrlFromJob(job, name);
           debug(`Archive ready. Prompting for download now (${url})`);
-          fileDownloadToast.options.downloadLink = url;
+          fileDownloadToast.options.archiveLink = url;
```
Comment (sergiofenoll) [inline app/components/agenda/agenda-header/agenda-actions.js]: These changes won't work. This here is using the `Utils::Toaster::DownloadFileToast` component, which expects the field to be called `downloadLink`.

**Review by sergiofenoll — CHANGES_REQUESTED** [review]
Undo these changes. Given the backend changes, the archive will still come from the regular file service and thus doesn't need a different download link.

**Comment by ValenberghsSven** [comment]
Changed as requested.
No changes needed to frontend with new (better) approach

**Review by sergiofenoll — APPROVED** [review]

**Comment by ValenberghsSven** [comment]
nothing to merge, not pushing commits

**Pull request closed without merging**